requirements.txt

```
@@ -1,2 +1,2 @@
 pollination-honeybee-radiance>=0.7.3
```

Hi @MingboPeng, this change should fix it.

```suggestion
pollination-honeybee-radiance>=0.7.4
```